.github/labeler.yml

```diff
@@ -0,0 +1,39 @@
+algorithms:
+  - src/pymortests/algorithms/*
+  - src/pymor/algorithms/*
+
+buildsystem:
+  - .ci/*
+  - .github/*
+  - .binder/*
+  - Makefile
+  - azure-pipelines.yml
+  - requirements*
+  - pyproject.toml
+  - MANIFEST.in
+  - dependencies.py
+  - setup.*
+  - versioneer.py
+
+bindings:
+  - src/pymor/bindings/*
+
+builtin-models:
+  - src/pymor/discretizers/*
+  - src/pymor/playground/discretizers/*
+  - src/pymortests/model.py
+  - src/pymortests/fixtures/model.py
+  - src/pymor/models/*
+
+docs:
+  - docs/*
+  - notebooks/*
+
+testing:
+  - src/pymortests/*
```

Comment:
This would add the 'testing' tag to every PR that implements tests together with the new feature. I do not think that this alone should be reason to add the tag.

Reply:
So you'd say the testing label is only appropriate if a PR _exclusively_ adds/changes tests? I do not think that's a good way to divide this.

Reply:
Basically, yes. Otherwise, this label would be assigned to every PR that introduces a new features or changes an API.